flutter-candidate.txt

```diff
@@ -1 +1 @@
-3.19.0-14.0.pre
+3.19.0-15.0.pre.41
```

Comment:
@derekxu16 ahh, if you use Flutter from candidate branches, you'll have to wait until the next flutter/flutter -> Google roll

Reply:
Oh, I see. I'll wait for that then.